### PR TITLE
ci(cypress): pass `is_click_to_pay_enabled` field in create business profile call

### PR DESCRIPTION
**Pull request by arindam-sahoo**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [x] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

This PR adds the `is_click_to_pay_enabled` field to the Cypress fixture for the Business Profile creation flow. This field was recently added to the flow but was missing in the `business_profile.json` fixture, which is required to ensure proper test coverage for scenarios where `is_click_to_pay_enabled` is either enabled or disabled.
### Changes made:
- Added the `is_click_to_pay_enabled` field to the `business_profile.json` fixture file.

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context

This change is required because the `is_click_to_pay_enabled` field was missing in the Cypress fixture.

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [ ] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: cypress-tests-v2/cypress/fixtures/business_profile.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
       "payment_created_enabled": true,
       "payment_succeeded_enabled": true,
       "payment_failed_enabled": true
-    }
+    },
+    "is_click_to_pay_enabled": false
```
Comment (SanchithHegde) [inline cypress-tests-v2/cypress/fixtures/business_profile.json:31]: Did we necessarily have to pass this field, or does the application take this field to be `false` if we don't pass it?

Comment (arindam-sahoo) [reply cypress-tests-v2/cypress/fixtures/business_profile.json:31]: If we don't pass it, it is not taking this field as `false`. It shows that the field is missing and the message goes like the following
Business profile create call failed with status 400 and message: "Json deserialize error: missing field `is_click_to_pay_enabled`

I am attaching a screenshot I took earlier
image

Comment (Narayanbhat166) [reply cypress-tests-v2/cypress/fixtures/business_profile.json:31]: @sahkal can you please fix this?

**Review by Narayanbhat166 — APPROVED** [review]

File: cypress-tests-v2/cypress/fixtures/business_profile.json (hunk 1 above)
Comment (sahkal) [reply cypress-tests-v2/cypress/fixtures/business_profile.json:31]: @Narayanbhat166 

https://github.com/juspay/hyperswitch/pull/6935

**Review by pixincreate — APPROVED** [review]

**Comment by pixincreate** [comment]
can you attach an issue to this pr?
also, do run the entire test suite once and paste the screenshot

**Comment by SanchithHegde** [comment]
I don't think this "fix" is required, the profile create v2 API should still work without the `is_click_to_pay_enabled` field being explicitly passed, as of the `2024.12.24.0` tag.

Closing this PR.

**Pull request closed without merging**